x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/connector/syncjob/action/PostConnectorSyncJobAction.java

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.application.connector.syncjob.action;
+
+import org.elasticsearch.ElasticsearchParseException;
+import org.elasticsearch.action.ActionRequest;
+import org.elasticsearch.action.ActionRequestValidationException;
+import org.elasticsearch.action.ActionResponse;
+import org.elasticsearch.action.ActionType;
+import org.elasticsearch.common.Strings;
+import org.elasticsearch.common.bytes.BytesReference;
+import org.elasticsearch.common.io.stream.StreamInput;
+import org.elasticsearch.common.io.stream.StreamOutput;
+import org.elasticsearch.common.xcontent.XContentHelper;
+import org.elasticsearch.xcontent.ConstructingObjectParser;
+import org.elasticsearch.xcontent.ParseField;
+import org.elasticsearch.xcontent.ToXContentObject;
+import org.elasticsearch.xcontent.XContentBuilder;
+import org.elasticsearch.xcontent.XContentParser;
+import org.elasticsearch.xcontent.XContentParserConfiguration;
+import org.elasticsearch.xcontent.XContentType;
+import org.elasticsearch.xpack.application.connector.syncjob.ConnectorSyncJobTriggerMethod;
+import org.elasticsearch.xpack.application.connector.syncjob.ConnectorSyncJobType;
+
+import java.io.IOException;
+import java.util.Objects;
+
+import static org.elasticsearch.action.ValidateActions.addValidationError;
+import static org.elasticsearch.xcontent.ConstructingObjectParser.constructorArg;
+import static org.elasticsearch.xcontent.ConstructingObjectParser.optionalConstructorArg;
+
+public class PostConnectorSyncJobAction extends ActionType<PostConnectorSyncJobAction.Response> {
+
+    public static final PostConnectorSyncJobAction INSTANCE = new PostConnectorSyncJobAction();
+
+    public static final String NAME = "cluster:admin/xpack/connector/sync_job/post";
```

Comment:
Why is this `cluster:admin/xpack/connector/sync_job/post` and not something like `cluster:admin/xpack/connector/post_sync_job`?

Reply:
I've checked how the constants are named inside [Constants.java](https://github.com/elastic/elasticsearch/blob/main/x-pack/plugin/security/qa/operator-privileges-tests/src/javaRestTest/java/org/elasticsearch/xpack/security/operator/Constants.java) and I found many instances, which put the `post` (or the action in general) after the resource (in this case `sync_job`):

See [this part in Constants](https://github.com/elastic/elasticsearch/blob/main/x-pack/plugin/security/qa/operator-privileges-tests/src/javaRestTest/java/org/elasticsearch/xpack/security/operator/Constants.java#L26-L42) for example. But I don't hold a strong opinion here. Do we've a best practice for the ent search action names?

Reply:
OK that's fair, thanks.